release/README.md

```diff
@@ -6,11 +6,52 @@ binaries are now reproducible, allowing developers to build the binary on
 distinct machines, and end up with a byte-for-byte identical binary. However,
 this wasn't _fully_ solved in `go1.13`, as the build system still includes the
 directory the binary is built into the binary itself. As a result, our scripts
-utilize a work around needed until `go1.13.2`.  
+utilize a work around needed until `go1.13.2`.
```

Comment:
One other thing related to this, is that all the releases should note which Go version was used to build the release so people can properly reproduce the binaries if they wish.

Reply:
Should have resolved this, placed notes in both the section for maintainers and the description up top